VideoLocker/src/androidTest/java/org/edx/mobile/test/http/ApiTests.java

```diff
@@ -111,6 +111,20 @@ public void testHandouts() throws Exception {
         print(model.handouts_html);
     }
 
+    public void testChannelId() throws Exception {
+        if( shouldSkipTest ) return;
```

Comment:
@aleffert  sure. we may need to create a separate project for testing involving network connection